### PR TITLE
GPC-NONE: Update links in schema.html.md.erb

### PR DESCRIPTION
**Pull request by olevett**
To clarify they're links

### CODE REVIEW

**Comment by galund** [comment]
I think I'd like to nuke this page as there are links to the schema files from the data model page.

**Comment by olevett** [comment]
Okay, sure :)

**Comment by olevett** [comment]
A link to this page went out to DWP recently, so I'll refrain from removing it completely, but have hidden it from the sidebar and made it reference the other related pages

File: techdocs/source/schema.html.md.erb [hunk 1]
```diff
@@ -1,10 +1,9 @@
...
 weight: 99
+hide_in_navigation: true
 ---
 
 # Schema
 
-We currently publish our event schema on the DI Identity Vocab pages, notably
-- [Death Registered](https://alphagov.github.io/di-identity-vocab/v1/json-schemas/DeathRegisteredJWT.json)
-- [Death Registration Updated](https://alphagov.github.io/di-identity-vocab/v1/json-schemas/DeathRegistrationUpdatedJWT.json)
+Our schema is referenced as part of the data model [here](https://docs.life-events.dev.account.gov.uk/data-model.html#classes-and-types)
```
Comment (galund) [inline techdocs/source/schema.html.md.erb:9]:
```suggestion
Our schema is referenced as part of the [data model](data-model.html#classes-and-types)
```

**Review by galund — APPROVED** [review]

**Pull request merged**